README.md

```diff
@@ -1,33 +1,44 @@
-# Godot Kotlin JVM
+# Godot Kotlin/JVM
 
-# Kotlin/JVM binding for the Godot Game Engine
+## Kotlin/JVM binding for the Godot Game Engine
 
 <p align="center">
 <img src=".README/logo.png" width="30%" height="30%">
 </p>
 
 ## Overview
 
-This is a **Kotlin** language binding for the [**Godot**](https://godotengine.org/) game engine. It is built as a module (like the C# binding) to interact with **Godot**'s core internally. The binding provides you Godot API's as Kotlin classes, so you can write your game logic completely in Kotlin. Your code will be compiled into a .jar which is then executed by an embedded JVM, so you don't have to worry that your users have Java installed. It's already embedded in your game executable.
-You also don't have to worry about any binding logic. Just write your game scripts like you would for [GDScript](https://docs.godotengine.org/en/3.1/getting_started/scripting/gdscript/gdscript_basics.html) or [C#](https://docs.godotengine.org/en/3.1/getting_started/scripting/c_sharp/) but with all the syntactic sugar of Kotlin.
+This is a **Kotlin** language binding for the [**Godot**](https://godotengine.org/) game engine.
+It is built as a module (like the C# binding) to interact with **Godot**'s core internally.
+The binding provides you Godot API's as Kotlin classes, so you can write your game logic completely in Kotlin.
+Your code will be compiled into a .jar which is then executed by an embedded JVM, so you don't have to worry that your users have Java installed.
```

Comment:
Suggestion:  
We have two ways to provide java executable:  
- Classic jar
- Dynamic library using graalvm native image